src/modules/getScore.js

```diff
@@ -0,0 +1,16 @@
+import requestAPI from './post.js';
+import displayScores from './displayScore.js';
+
+const sortScores = (scores) => scores.sort((a, b) => b.score - a.score);
+
+async function getScores() {
+  const data = await requestAPI('GET');
+  const scores = data.result.map(({ user, score }) => ({ user, score: parseInt(score, 10) }));
+  displayScores(sortScores(scores));
+  return sortScores(scores);
+}
+
+async function postScore(user, score) {
+  await requestAPI('POST', { user, score });
+}
```

Comment:
- [ ] It will be better to use arrow functions instead of the function keyword as stated in the requirements of the project and also  consider changing the rest functions to arrow functions syntax

Reply:
Thank you very much for giving me the right advice. Now I will submit my project again changing all my functions to the arrow function.